watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -999,6 +1063,27 @@ def rule_pressure_osm_phase(b):
 
         self.eq_pressure_osm_phase = Constraint(rule=rule_pressure_osm_phase)
 
+    def _electrical_conductivity_phase(self):
+        self.electrical_conductivity_phase = Var(
+            self.params.phase_list,
+            initialize=0.1,
+            units=pyunits.ohm**-1 * pyunits.meter**-1,
+            doc="Electrical conductivity",
+        )
+
+        def rule_electrical_conductivity_phase(b):
+            return b.electrical_conductivity_phase["Liq"] == sum(
+                Constants.faraday_constant
+                * abs(b.charge_comp[j])
+                * b.electrical_mobility_comp[j]
+                * b.conc_mol_phase_comp["Liq", j]
+                for j in self.params.ion_set
+            )  # maybe revisit for other emprical calculation or non-ideal situation
```

Comment:
The new parameter for electrical mobility includes both `ion_set` and `solute_set`, but we only iterate through `ion_set` here. (See comment above about the new parameter).